### PR TITLE
Context Fixture

### PR DESCRIPTION
**Pull request by jvmlet**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -70,6 +70,25 @@ public class TestBaseSample  :
 <!-- endSnippet -->
 
 
+## xunit Fixture
+
+In addition to `XunitContextBase` class approach, one is also possible to use  `IContextFixture` to gain access to `XunitContext` :
+```cs
+public class UsingClassFixture : IContextFixture
```
Comment (SimonCropp) [inline readme.md:77]: can u make this a snippet like the other code in the readme. 

see https://github.com/SimonCropp/MarkdownSnippets

File: src/src/Tests/UsingClassFixture.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
+using Xunit.Fixture;
+
+namespace Tests;
```
Comment (SimonCropp) [inline src/src/Tests/UsingClassFixture.cs:3]: kill the namespace

**Comment by jvmlet** [comment]
@SimonCropp ?

**Comment by SimonCropp** [comment]
deploying now

**Pull request merged**